modules/config/pages/eap-config.adoc

```diff
@@ -1,11 +1,159 @@
+
+. To verify that the JBoss EAP server is utilizing the new, copied configuration files, update the port offset. You can run multiple instances of EAP, by using the port offset to avoid port conflicts.
+
+. Update the new configuration file by editing the standalone.xml file to verify that JBoss EAP is using the configuration files in the new directory. Open the `/opt/standalone-running/configuration/standalone.xml` file with a text editor.
+
+. In the standard-sockets socket binding group, change the default value of the port-offset attribute from 0 to 10000. This adds 10,000 to every port number used in the standard-sockets binding group.
++
+[subs="+quotes,+macros"]
+----
+[vagrant@server ~]$ sudo vi /opt/standalone-running/configuration/standalone.xml
+...
+<socket-binding-group name="standard-sockets" default-interface="public" port-offset="${jboss.socket.binding.port-offset:10000}">
+...
+----
+
+Save your changes to the standalone.xml file, and close the text editor.
+
+* Run and test the JBoss EAP Server.
+
+. Run the following command to start the JBoss EAP server by using the standalone.sh script in the original JBoss EAP installation, but using the new configuration files:
++
+[subs="+quotes,+macros"]
+----
+[vagrant@server ~]$ cd /opt/EAP-8.0.0/bin
+[vagrant@server bin]$ sudo ./standalone.sh \
+-Djboss.server.base.dir=/opt/standalone-running/
+----
+
+. The server starts up successfully with an output similar to the following:
++
+[subs="+quotes,+macros"]
+----
+10:24:24,663 INFO  [org.jboss.ws.common.management] (MSC service thread 1-4) JBWS022052: Starting JBossWS 7.0.0.Final-redhat-00001 (Apache CXF 4.0.0.redhat-00002) 
+10:24:24,666 INFO  [org.jboss.as.server.deployment.scanner] (MSC service thread 1-2) WFLYDS0013: Started FileSystemDeploymentService for directory /opt/standalone-running/deployments
+10:24:24,793 INFO  [org.jboss.as.server] (Controller Boot Thread) WFLYSRV0212: Resuming server
+10:24:24,797 INFO  [org.jboss.as] (Controller Boot Thread) WFLYSRV0060: Http management interface listening on http://127.0.0.1:19990/management
+10:24:24,797 INFO  [org.jboss.as] (Controller Boot Thread) WFLYSRV0051: Admin console listening on http://127.0.0.1:19990
+----
+
+* Navigate to http://localhost:18080 to see the JBoss EAP welcome page with the new port offset.
+
+* Navigate to http://localhost:19990 to see the JBoss EAP management console, which runs with the same port offset.
+
+* Explore the contents of the directory `/opt/standalone-running` in a new terminal window:
++
+[subs="+quotes,+macros"]
+----
+[vagrant@server ~]$ ls /opt/standalone-running/
+configuration  data   deployments  lib  log  tmp
+----
+
+Notice the three data, log, and tmp new directories . These directories are automatically created when the JBoss EAP server starts.
+
+* Clean Up
+
+Stop the running instance of JBoss EAP that was started in the previous step. Press `Ctrl+C` in the terminal window in which the server is running.
```

Comment:
I wonder if this is right section to talk about the port binding flag - that is, how to bind and run EAP on a NIC which has a public IP instead of localhost. Maybe one more lab in this section - a short one showing how to bind different IPs for HTTP traffic and management. See ./standalone.sh --help and product docs for the various other flags. See what else is important and can be covered in this section